app/components/works/buttons_component.html.erb

```diff
@@ -23,6 +23,8 @@
     <%= form.submit 'Save as draft', class: 'btn btn-primary', id: 'save-draft-button',
                                      data: { action: 'unsaved-changes#allowFormSubmission' } %>
     <%= form.submit submit_button_label, class: 'btn btn-primary',
+                                         disabled: disabled?,
```

Comment:
This seemed to work to avoid the button having `disabled-""` which actually makes it disabled.